File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT1001U01LayoutBarCodeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroOUT1001U01LayoutBarCodeInfo
    {
        private String _bunho;
        private String _suname;
        private String _sex;
        private String _birth;
        private String _bunho2;

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Sex
        {
            get { return this._sex; }
            set { this._sex = value; }
        }

        public String Birth
        {
            get { return this._birth; }
            set { this._birth = value; }
        }

        public String Bunho2
        {
            get { return this._bunho2; }
            set { this._bunho2 = value; }
        }

        public NuroOUT1001U01LayoutBarCodeInfo() { }

        public NuroOUT1001U01LayoutBarCodeInfo(String bunho, String suname, String sex, String birth, String bunho2)
        {
            this._bunho = bunho;
            this._suname = suname;
            this._sex = sex;
            this._birth = birth;
            this._bunho2 = bunho2;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT1001U01LayoutBarCodeInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroOUT1001U01LayoutBarCodeInfo {
	 private String bunho;
     private String suname;
     private String sex;
     private String birth;
     private String bunho2;
     

	public NuroOUT1001U01LayoutBarCodeInfo(String bunho, String suname,
			String sex, String birth, String bunho2) {
		super();
		this.bunho = bunho;
		this.suname = suname;
		this.sex = sex;
		this.birth = birth;
		this.bunho2 = bunho2;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public String getBunho2() {
		return bunho2;
	}
	public void setBunho2(String bunho2) {
		this.bunho2 = bunho2;
	}
	public String getBirth() {
		return birth;
	}
	public void setBirth(String birth) {
		this.birth = birth;
	}
     
     
}
